src/ext/default.h

```diff
@@ -2,6 +2,7 @@
 #define __EXT_DEFAULT_H__
 #include "redisearch.h"
 
+#define SYNONYMS_EXPENDER_NAME "SYN"
```

Comment:
The default expander should do both synonyms and stems - or rather wrap them both. 

Optionally the user can specify `EXPANDER SBSTEM|SYNONYM` to invoke just one, or `EXPANDER DEFAULT` to invoke the default. 

Please update the docs with the options.